release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2578,6 +2578,8 @@ If NMEA sentences are lost on their way to the e810 NIC, the T-GM cannot synchro
 A proposed fix is to report a `FREERUN` event when the NMEA string is lost.
 (link:https://issues.redhat.com/browse/OCPBUGS-19838[*OCPBUGS-19838*])
 
+* Currently, the YAML tab of some pages in the web console will stop unexpectedly on some browsers when the multicluster engine for Kubernetes operator (MCE) is installed. An error notification "`TypeErrorDescription:e` is undefined`" is displayed. https://issues.redhat.com/browse/OCPBUGS-29812[*OCPBUGS-29812*]
```

Comment:
The error message is a little different, can we update? attaching a screenshot for reference
<img width="1385" alt="Screenshot 2024-02-23 at 3 39 56 PM" src="https://github.com/openshift/openshift-docs/assets/12692381/f714aaa4-8ae7-4cf3-9678-ac9bc93aa6c7">

Reply:
Thank you for the screenshot! I have updated the note.